doc: autopts: Remove duplication of getting started guide (#39137)
zephyrproject-rtos/zephyr: doc/guides/bluetooth/autopts/autopts-win10.rst

.. code-block::

   dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart

>    wsl --set-default-version 1

Reviewer: I'm curious about this line: Why should the version be 1 instead of 2?

   :width: 800
   :align: center

When finished, perform the Ubuntu setup from `Getting Started Guide <https://docs.zephyrproject.org/latest/getting_started/index.html>`_,

> but skip west init and west update commands, as you can reach downloaded before
> Zephyr source code at mounted Windows folder /mnt/c/Users/<user>/zephyrproject.

Reviewer: ```suggestion
but skip west init and west update commands, as you can use the previously downloaded
Zephyr source code at mounted Windows folder /mnt/c/Users/<user>/zephyrproject.
```
